sender/TelegramSender.py

```diff
@@ -18,45 +21,47 @@ def __init__(
         self.token = token
         self.chat_id = chat_id
         self.alert_chat_id = alert_chat_id
-        self.retry_interval = retry_interval
 
         self.bot_emoji = bot_emoji
         self.top_emoji = top_emoji
         self.news_emoji = news_emoji
 
-        self.logger = logging.getLogger("telegram-sender")
+        self.telegram_executor = ThreadPoolExecutor(max_workers=3)
 
-        # Initialize telegram bot
-        try:
-            self.bot = Bot(token=token)
-        except Exception as e:
-            self.logger.error("Error initializing Telegram bot. Exception: %s.", e)
-            quit()
+        self.request = Request(con_pool_size=30)
```

Comment:
Is this supposed to be changed to 3 to match with the change of max_workers = 3?